tendrl/integrations/gluster/check_cluster_status.py

```diff
@@ -0,0 +1,29 @@
+import etcd
+
+from tendrl.commons.utils import etcd_utils
+
+
+def run():
+    try:
+        clusters = etcd_utils.read("/clusters")
+    except etcd.EtcdKeyNotFound:
+        return
+
+    # This logic only runs on node with tag `tendrl/monitor` (tendrl server)
+    # So its done checked for all the clusters and tries to set cluster
+    # status as `unhealthy` if the status field is missing (due to TTL)
+    for cluster in clusters.leaves:
+        int_id = cluster.key.split('/')[-1]
+        try:
+            if cluster.is_managed == "yes":
```

Comment:
I am not sure how you have tested this. etcd_utils.read returns EtcdResult. you cannot do a "cluster.is_managed" directly. Thoughts?

Reply:
Updated the changes